### PR TITLE
Overlay fixes

### PR DESCRIPTION
**Pull request by calderon**
### What is the goal?

Currently, to create a new object Overlay needs an empty object as argument, and it doesn't need that.

Also, styles are not correct.

- [ ] It passses the `jt-frontend` lint commands (`--lint-js`, `--lint-less`, `--lint-coffee`, ...)
- [ ] The `README.md` has been updated accordingly
- [ ] **Issue:** https://jobandtalent.atlassian.net/browse/FRONT-123

### How is being implemented?

Just a minor change in the controller, also styles are modified.

### Reviewers

@jobandtalent/frontend

### CODE REVIEW

**Comment by montiu** [comment]
👍  after resolve conflicts

File: src/component/overlay/main.less [hunk 1]
```diff
@@ -1,6 +1,15 @@
-@import '../../stylesheets/colors.less';
-.jt {
-  .overlay {
-    background-color: @dark-text;
+@import '../../stylesheets/colors';
+
+.page_overlay {
```
Comment (tomasperezv) [inline src/component/overlay/main.less]: Don't you want to keep the namespace `.jt`?

Comment (calderon) [inline src/component/overlay/main.less]: I think this has never been used, not for us at least

Comment (tomasperezv) [inline src/component/overlay/main.less]: @calderon right, but we added it for a reason, any problem in keeping the namespace?

Comment (calderon) [inline src/component/overlay/main.less]: @tomasperezv Not a real problem. It's not being currently used, also we don't think we will have conclicts

**Comment by tomasperezv** [comment]
:+1:

**Pull request merged**